src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoOffhand.kt

```diff
@@ -63,7 +63,7 @@ internal object AutoOffhand : Module(
     private val checkCACrystal by setting("Check Crystal Aura C", false, { type == Type.CRYSTAL && offhandCrystal })
 
     // General
-    private val priority by setting("Priority", Priority.HOTBAR)
+    private val priorityInv by setting("Priority", Priority.HOTBAR)
```

Comment:
I'd much rather the abstract inherited `priority` setting be called something more verbose then making people rename their modules